hathor/transaction/resources/block_at_height.py

```diff
@@ -75,8 +75,29 @@ def render_GET(self, request: 'Request') -> bytes:
             })
 
         block = self.manager.tx_storage.get_transaction(block_hash)
-
         data = {'success': True, 'block': block.to_json_extended()}
+
+        # include_transactions parameter is optional and must be an integer
+        try:
+            include_transactions = parse_int(args.get('include_transactions', 0))
+        except ValueError as e:
+            return json_dumpb({
+                'success': False,
+                'message': f'Failed to parse \'include_transactions\': {e}'
+            })
+
+        if include_transactions == 1:
+            tx_ids: list[str] = []
+            for tx in block.iter_transactions_in_this_block():
+                tx_ids.append(tx.hash.hex())
+            data['tx_ids'] = tx_ids
```

Comment:
Should we keep the fields the same? So having `transactions` for both, instead of `tx_ids` in case it's just the ids.

Reply:
I used separate keys to facilitate parsing since they have different formas.